backend/unimeet/views.py

```diff
@@ -19,7 +16,7 @@ def get_schools(request):
 @csrf_exempt
 def signup(request):
     signup_parameters = json.loads(request.body.decode('utf-8'))
-    email = signup_parameters['email']
+    email = signup_parameters.get('email')
```

Comment:
Why are these changes necessary?

Reply:
In case the user does not pass any parameters, we don't want it to crash with an exception (status code 500) – instead it should fail gracefully by behaving as if the email was set to `None` (status code 400).